src/test/fixtures/spdx-not-allowed-ignore/expected/build/reports/licensee/validation.txt

```diff
@@ -0,0 +1,2 @@
+com.example:example:1.0.0
```

Comment:
What's the deal with this? Is this a log or an output? I'm confused as to why this is expected for the ignore case.

Reply:
It's a full log of all artifacts and their status. So even in the case when everything is allowed and your build passes you'll have entries in here like:
```
com.example:example:1.0.0
 - SPDX identifier 'Apache-2.0' allowed
```

The new property only controls the build action which is taken when a violation is found. It does not affect this file at all.

Reply:
I'll add a note to the kdoc